ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -30,7 +30,7 @@ parameters:
 - name: OPENSTACK_EXTERNAL_NETWORK
   value: external
 - name: OPENSTACK_FLAVOR
-  value: m1.s2.xlarge
+  value: c2.s2.xlarge
```

Comment:
It would be nice to back findings with Prometheus data.
https://promecieus.svc.ci.openshift.org/